<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calculator</class>
 <widget class="QMainWindow" name="Calculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>259</width>
    <height>371</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icon.qrc">
    <normaloff>:/new/main_icon/res/calculator_icon.png</normaloff>:/new/main_icon/res/calculator_icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="result">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>241</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	qproperty-allignment: 'AllignVCenter | AllignRight';
	border-bottom =&quot; 1px solid gray;
}

background-color: white;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="margin">
     <number>2</number>
    </property>
    <property name="indent">
     <number>7</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_AC">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(215, 215, 215);
	border: 1px solid gray;
}
QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1:0, x2: 0, y2: 1, stop: 0 #BEBEBE, 
										stop:1 #D7D7D7);
}</string>
    </property>
    <property name="text">
     <string>AC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sgn">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>60</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(215, 215, 215);
	border: 1px solid gray;
}
QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1:0, x2: 0, y2: 1, stop: 0 #BEBEBE, 
										stop:1 #D7D7D7);
}</string>
    </property>
    <property name="text">
     <string>+/-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_percent">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(215, 215, 215);
	border: 1px solid gray;
}
QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1:0, x2: 0, y2: 1, stop: 0 #BEBEBE, 
										stop:1 #D7D7D7);
}</string>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_div">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>60</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(255, 151, 57);
	color: white;
	border: 1px solid gray;
}
QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1:0, x2: 0, y2: 1, stop: 0 #FF7832, 
										stop:1 #FF9739);
}</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>120</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid gray;
}
QPushButton:pressed {
	background-color: qlineargradient(x1:0, y1: 0, x2: 0, y2: 1,
										stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid gray;
}
QPushButton:pressed {
	background-color: qlineargradient(x1:0, y1: 0, x2: 0, y2: 1,
										stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_bazm">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>120</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(255, 151, 57);
	color: white;
	border: 1px solid gray;
}
QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1:0, x2: 0, y2: 1, stop: 0 #FF7832, 
										stop:1 #FF9739);
}</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>120</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid gray;
}
QPushButton:pressed {
	background-color: qlineargradient(x1:0, y1: 0, x2: 0, y2: 1,
										stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>180</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid gray;
}
QPushButton:pressed {
	background-color: qlineargradient(x1:0, y1: 0, x2: 0, y2: 1,
										stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid gray;
}
QPushButton:pressed {
	background-color: qlineargradient(x1:0, y1: 0, x2: 0, y2: 1,
										stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_minus">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>180</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(255, 151, 57);
	color: white;
	border: 1px solid gray;
}
QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1:0, x2: 0, y2: 1, stop: 0 #FF7832, 
										stop:1 #FF9739);
}</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>180</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid gray;
}
QPushButton:pressed {
	background-color: qlineargradient(x1:0, y1: 0, x2: 0, y2: 1,
										stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>240</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid gray;
}
QPushButton:pressed {
	background-color: qlineargradient(x1:0, y1: 0, x2: 0, y2: 1,
										stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid gray;
}
QPushButton:pressed {
	background-color: qlineargradient(x1:0, y1: 0, x2: 0, y2: 1,
										stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_plus">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>240</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(255, 151, 57);
	color: white;
	border: 1px solid gray;
}
QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1:0, x2: 0, y2: 1, stop: 0 #FF7832, 
										stop:1 #FF9739);
}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>240</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid gray;
}
QPushButton:pressed {
	background-color: qlineargradient(x1:0, y1: 0, x2: 0, y2: 1,
										stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_0">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid gray;
}
QPushButton:pressed {
	background-color: qlineargradient(x1:0, y1: 0, x2: 0, y2: 1,
										stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_dot">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>300</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(215, 215, 215);
	border: 1px solid gray;
}
QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1:0, x2: 0, y2: 1, stop: 0 #BEBEBE, 
										stop:1 #D7D7D7);
}</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_equ">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>300</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(255, 151, 57);
	color: white;
	border: 1px solid gray;
}
QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1:0, x2: 0, y2: 1, stop: 0 #FF7832, 
										stop:1 #FF9739);
}</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
